### PR TITLE
Fix conformance test for canton dev

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh [hunk 1]
```diff
@@ -8,24 +8,7 @@ set -o pipefail
...
-
-export UNIQUE_CONTRACT_KEYS="$(rlocation com_github_digital_asset_daml/ledger/ledger-api-test-tool-on-canton/unique-contract-keys.conf)"
-if [[ -f ${UNIQUE_CONTRACT_KEYS} ]]; then
-  command+=("--config=${UNIQUE_CONTRACT_KEYS}")
-fi
-
-export ENABLE_FASTER_PRUNING="$(rlocation com_github_digital_asset_daml/ledger/ledger-api-test-tool-on-canton/enable-faster-pruning.conf)"
-if [[ -f ${ENABLE_FASTER_PRUNING} ]]; then
-  command+=("--config=${ENABLE_FASTER_PRUNING}")
-fi
```
Comment (remyhaemmerle-da) [inline ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh:28]: dead code

File: ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh [hunk 2]
```diff
@@ -8,24 +8,7 @@ set -o pipefail
...
+command=("/home/remy/work/canton/community/app/target/scala-2.13/canton-open-source-2.4.0-SNAPSHOT.jar" daemon "$@")
```
Comment (skisel-da) [inline ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh:11]: accidential commit?

Comment (remyhaemmerle-da) [reply ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh:11]: Yes :S

Comment (remyhaemmerle-da) [reply ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh]: fixed

File: ledger/ledger-api-test-tool-on-canton/bootstrap-dev.canton [hunk 3]
```diff
@@ -0,0 +1,17 @@
+nodes.local.start
+
+test_domain.service.set_mediator_deduplication_timeout(6.minutes)
```
Comment (rgugliel-da) [inline ledger/ledger-api-test-tool-on-canton/bootstrap-dev.canton:3]: I asked Matthias: values are sane

File: ledger/ledger-api-test-tool-on-canton/canton-dev.conf [hunk 4]
```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+_shared {
```
Comment (rgugliel-da) [inline ledger/ledger-api-test-tool-on-canton/canton-dev.conf:4]: We could also apply a second conf on top of the first to reduce duplication but if you just want this to be quickly merged, fine for me.

**Comment by rgugliel-da** [comment]
So I guess this cannot pass CI before we have the canton snapshot that depends on the future daml snapshot ?

**Comment by remyhaemmerle-da** [comment]
> So I guess this cannot pass CI before we have the canton snapshot that depends on the future daml snapshot ?

It should pass. However to enable the test on LF 1.15  we need an update of canton. 

To activate LF 1.15 we need to use `preview` as `lf_version` in `conformance-test-dev-domain`,

**Review by rgugliel-da — APPROVED** [review]
Canton related changes LGTM

**Review by skisel-da — APPROVED** [review]

**Review by akrmn — APPROVED** [review]
LGTM

**Pull request merged**